Assembler.Base/FrameHandlers/FinalFrameHandler.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.Enums;
+
+namespace Assembler.Base.FrameHandlers
+{
+    public class FinalFrameHandler<TFrame, TMessageInAssembly> : BaseFrameHandler<TFrame, TMessageInAssembly>
```

Comment:
See comments of InitialFrameHandler and apply

Reply:
f

Reply:
f